migrations/src/postDeploymentSetup.js

```diff
@@ -45,36 +42,21 @@ module.exports = async () => {
   if (NETWORK !== 'mainnet') {
     // Add new instance of StakeManager contract & Deployer address as Minter
 
-    const initialSupply = await schellingCoin.INITIAL_SUPPLY();
-
-    const mintableSupply = (BigNumber.from(10).pow(BigNumber.from(26))).mul(BigNumber.from(6));
-    const deployerBalance = BigNumber.from(await schellingCoin.balanceOf(signers[0].address));
-    const deployerSupply = BigNumber.from(initialSupply).sub(BigNumber.from(deployerBalance));
-
-    if (SCHELLING_COIN_ADDRESS !== '') {
-      // if previous instances of Schelling Coin is reused again and again,
-      // then initial balance will get depleted, thus intial tokens minting is needed,
-      // each time Schelling Coin instance is reused
-      await schellingCoin.addMinter(signers[0].address);
+    const initialSupply = (BigNumber.from(10).pow(BigNumber.from(26))).mul(BigNumber.from(6));
```

Comment:
Please rename this to the appropriate name
This is not initialsupply. 
Its The Supply we are passing to StakeManager for rewards.

In our last comment initialSupply was making sense( in deployment , constructor, ) as it was initial supply

Reply:
yeah I will change this